From 1f60a039406af04116d154db27be8f71fe974342 Mon Sep 17 00:00:00 2001
From: Sebastian Reichel <sebastian.reichel@collabora.com>
Date: Tue, 25 Jul 2023 15:47:18 +0200
Subject: [PATCH 63/98] arm64: defconfig: enable Synopsys AHCI SATA support

Enable support for the DesignWare AHCI Host Controller. It is used
by recent Rockchip SoCs.

Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -299,6 +299,7 @@ CONFIG_SATA_AHCI=y
 CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_AHCI_BRCM=m
 CONFIG_AHCI_CEVA=y
+CONFIG_AHCI_DWC=m
 CONFIG_AHCI_MVEBU=y
 CONFIG_AHCI_XGENE=y
 CONFIG_AHCI_QORIQ=y
